maro/simulator/scenarios/vm_scheduling/topologies/azure.2019.336k/config.yml

```diff
@@ -0,0 +1,27 @@
+# Initial buffer time budget.
+BUFFER_TIME_BUDGET: 0
+# The number of ticks to delay when agent decides not to assign VM.
+DELAY_DURATION: 1
+
+# Path of the vm table data.
+VM_TABLE: "~/.maro/data/vm_scheduling/.build/azure.2019.336k/vmtable.bin"
+
+# Path of the cpu readings file.
+CPU_READINGS: "~/.maro/data/vm_scheduling/.build/azure.2019.336k/vm_cpu_readings-file-1-of-195.bin"
+
+PROCESSED_DATA_URL: "https://marodatasource.blob.core.windows.net/vm-scheduling-azure/azure.2019.336k/azure.2019.336k.tar.gz"
+
+PM:
+  P1:
```

Comment:
P1 will be replaced by the SKU No. later?

Reply:
Yes